qtop.py

```diff
@@ -2315,27 +2313,46 @@ class InvalidScheduler(Exception):
         new_attrs = old_attrs[:]
 
     available_batch_systems = discover_qtop_batch_systems()
-
     stdout = sys.stdout  # keep a copy of the initial value of sys.stdout
     change_mapping = cycle([('queue_to_color', 'color by queue'), ('user_to_color', 'color by user')])
     h_counter = cycle([0, 1])
-
     viewport = Viewport()  # controls the part of the qtop matrix shown on screen
     max_line_len = 0
-
-
     check_python_version()
     initial_cwd = os.getcwd()
     logging.debug('Initial qtop directory: %s' % initial_cwd)
     CURPATH = os.path.expanduser(initial_cwd)  # where qtop was invoked from
     QTOPPATH = os.path.dirname(realpath(sys.argv[0]))  # dir where qtop resides
     HELP_FP = os.path.join(QTOPPATH, 'helpfile.txt')
     help_main_switch = [HELP_FP, ]  # output_fp is not yet defined, will be appended later
-    SAMPLE_FILENAME = 'qtop_sample_${USER}%(datetime)s.tar'
-    SAMPLE_FILENAME = os.path.expandvars(SAMPLE_FILENAME)
+    SAMPLE_FILENAME = os.path.expandvars('qtop_sample_${USER}%(datetime)s.tar')
+
+    return options, args, dynamic_config, old_attrs, new_attrs, available_batch_systems, stdout, change_mapping, \
+           h_counter, viewport, max_line_len, initial_cwd, CURPATH, QTOPPATH, HELP_FP, help_main_switch, SAMPLE_FILENAME
+
+if __name__ == '__main__':
+
+    options, \
+    args, \
+    dynamic_config, \
+    old_attrs, \
+    new_attrs, \
+    available_batch_systems, \
+    stdout, \
+    change_mapping, \
+    h_counter, \
+    viewport, \
+    max_line_len, \
+    initial_cwd, \
+    CURPATH, \
+    QTOPPATH, \
+    HELP_FP, \
+    help_main_switch, \
```

Comment:
nice. for readability, how about collapsing the above into 2 lines, to make it visually alike L2330?